### ISSUE TITLE
nullPointerException on setBuildStatus after build success

### ISSUE THREAD

**Issue by disseminate** [nishio-dens/bitbucket-pullrequest-builder-plugin#138]
Getting this error after a successful build.

```
Nov 16, 2017 7:25:14 PM hudson.model.listeners.RunListener report
WARNING: RunListener failed
java.lang.NullPointerException
        at bitbucketpullrequestbuilder.bitbucketpullrequestbuilder.BitbucketRepository.setBuildStatus(BitbucketRepository.java:153)
        at bitbucketpullrequestbuilder.bitbucketpullrequestbuilder.BitbucketBuilds.onCompleted(BitbucketBuilds.java:46)
        at bitbucketpullrequestbuilder.bitbucketpullrequestbuilder.BitbucketBuildListener.onCompleted(BitbucketBuildListener.java:36)
        at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:211)
        at org.jenkinsci.plugins.workflow.job.WorkflowRun.finish(WorkflowRun.java:664)
        at org.jenkinsci.plugins.workflow.job.WorkflowRun.access$1400(WorkflowRun.java:135)
        at org.jenkinsci.plugins.workflow.job.WorkflowRun$GraphL.onNewHead(WorkflowRun.java:985)
        at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.notifyListeners(CpsFlowExecution.java:1216)
        at org.jenkinsci.plugins.workflow.cps.CpsThreadGroup$3.run(CpsThreadGroup.java:407)
        at org.jenkinsci.plugins.workflow.cps.CpsVmExecutorService$1.run(CpsVmExecutorService.java:35)
        at hudson.remoting.SingleLaneExecutorService$1.run(SingleLaneExecutorService.java:112)
        at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

The bitbucket PR's status is still pending after this error.

**Comment by blakelloyd-coderus** [comment]
Am also receiving the same error, I believe #126 should fix this? Not sure on the timeframe of it being released though.

**Comment by CodeMonk** [comment]
@sazo Is this still happening?

**Comment by celicoo** [comment]
@CodeMonk yes.

**Comment by CodeMonk** [comment]
@celicoo Can you please provide logs for this different issue? It's surprising that you're hitting both of these issues. . .

**Comment by CodeMonk** [comment]
And, @celicoo @sazo Are you still seeing this behavior after #126 ?

**Comment by sazo** [comment]
I don't use this plug-in anymore - If i was you I wot use bitbucket source branch plug-in. It has it all :)

**Comment by CodeMonk** [comment]
Closing - prolly fixed in #126.
